reaktive/src/commonMain/kotlin/com/badoo/reaktive/utils/Utils.kt

```diff
@@ -7,12 +7,12 @@ internal fun handleSourceError(error: Throwable, onError: ((Throwable) -> Unit)?
         } else {
             try {
                 onError(error)
-            } catch (ignored: Throwable) {
-                reaktiveUncaughtErrorHandler(error)
```

Comment:
I think it may be worth stating here that it is an exception on trying to deliver an error result. In an effort to make debugging a bit clearer.

Reply:
Good point, fixed